manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -192,6 +192,9 @@ telemetry:
       #  meshEdgesReportingDuration: 500s
       #  disable_host_header_fallback: true
 
+    accessLog:
+      enabled: false
+      logWindowDuration: "43200s"
```

Comment:
can you add a comment that documents why this defaults to `12h` ?

Reply:
Thanks..Fixed

Reply:
This name is confusing `accessLog` feels like you are  enabling or disabling access log instead of `access_log_policy` rename to `accessLogFiltering`

Reply:
Access log filtering could be confused with native envoy access log filtering. In general, filter is a terrible term since it means so many things.

Reply:
AccessLogPolicy then?

Reply:
Ok that sounds good.
Put a comment online about what it will do.

Reply:
Thanks..Fixed